torch/export/_tracer.py

```diff
@@ -0,0 +1,615 @@
+import copy
```

Comment:
@angelayi I'm nitpicking the name here. I prefer to merge this file with export/__init__.py, but it's also okay to keep it in separate file, better name to me is `_trace.py`